keel-core/keel-core.gradle.kts

```diff
@@ -24,9 +24,10 @@ dependencies {
   api("com.fasterxml.jackson.module:jackson-module-kotlin")
   api("com.fasterxml.jackson.datatype:jackson-datatype-jsr310")
   api("de.huxhorn.sulky:de.huxhorn.sulky.ulid")
-  api("de.danielbechler:java-object-diff")
   api("com.netflix.spinnaker.kork:kork-artifacts")
 
+  implementation("de.danielbechler:java-object-diff")
```

Comment:
Made this not get exported so other modules can't directly reference types from it

Reply:
Actually can't do this (yet)